src/i18n/i18n.ts

```diff
@@ -0,0 +1,20 @@
+import i18n from 'i18next';
```

Comment:
If we import this file as `src/i18n/i18n` in other modules I think it should be renamed to `src/i18n/index.ts`.